Code/GraphMol/Depictor/Templates.h

```diff
@@ -46,23 +46,99 @@ class CoordinateTemplates {
     return m_templates[atom_count];
   }
 
- private:
-  CoordinateTemplates() {
-    // load templates into m_templates map by atom count
+  bool setRingSystemTemplates(const std::string& template_dir) {
```

Comment:
Should we move this implementation to a .cpp file?

Reply:
That makes sense to me -- lmk how this looks